src/components/nested_screens/search/StockInfoScreen.js

```diff
@@ -40,10 +41,24 @@ export default function StockInfoScreen(props) {
     { x: 8, y: 10 },
   ];
 
+  const stockData = {
+    recentPrice: 152.01,
+    open: 150.64,
+    high: 153.19,
+    low: 150.64,
+    yearlyHigh: 176.15,
+    yearlyLow: 124.17,
+    priceDifferenceRaw: 1.47,
```

Comment:
Messaged Jesse about this but current the raw price difference is being returned so he will need to add that to the API

Reply:
Alright thats ok. Was just trying to simulate the flow so its easier for Tim to integrate.